### ISSUE TITLE
Error no rows

### ISSUE THREAD

**Issue by franklupo** [douglasg14b/BetterConsoleTables#22]
Hi,
if you create a table without rows the ToString() method returns errors.

```c#
 var table = new Table(..);
....
table.AddRows(rows); // rows count = 0
var ret = table.ToString();

System.IndexOutOfRangeException
Index was outside the bounds of the array.
   at BetterConsoleTables.Table.ToString(Int32[] columnLengths)
   at BetterConsoleTables.Table.ToString()
```

**Comment by douglasg14b** [comment]
Is this on v1 or v2?

Additionally please post a full example.

**Comment by franklupo** [comment]
Version 1.1.2

**Comment by franklupo** [comment]
Example https://dotnetfiddle.net/deemYE

**Comment by douglasg14b** [comment]
This should now be fixed in v1.1.3

Please test in your environment, and close this issue if so.

**Comment by douglasg14b** [comment]
Closing this as there has been no reply and the problem should be resolved.
